hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -156,7 +157,7 @@ private void createMemberState(MemberStateImpl memberState,
         }
         memberState.setClients(serializableClientEndPoints);
         memberState.setName(instance.getName());
-
+        memberState.setMemberTime(currentTimeMillis());
```

Comment:
There is a [TimedMemberState#setTime](https://github.com/hazelcast/hazelcast/blob/85bc1f26d5f87e35f4cd03de07549257120acf04/hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberState.java#L77). Can we make use of it instead?